.github/scripts/run_tests.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -eo pipefail
+export PATH=/opt/conda/bin:$PATH
+
+# write log as github Action artifact
+mkdir artifacts
+echo run tests >> artifacts/test_artifact.log
+
+# run current pull request code
+singularity --version
+NXF_VER=20.03.0-edge nextflow run ./main.nf \
+       -profile singularity \
+       --directory $PWD/.github/data/ \
+       --illumina \
+       --prefix test
+cp .nextflow.log artifacts/
+
+# run upstream connor-lab fork
```

Comment:
This is going to end up as a PR in connor-lab, so this test doesn't make sense. Once connor-lab is tagged with releases, perhaps the test could compare to the previous release. For now you could checkout commit 42a79999f11304671b9d22e4959b0167b2130944 to compare against.

Reply:
ok, done.